apps/prairielearn/src/sync/dates.ts

```diff
@@ -1,6 +1,9 @@
 import { isFuture, isValid, parseISO } from 'date-fns';
 
-export function isAccessRuleAccessibleInFuture({ endDate }: { endDate?: string | null }) {
+/**
+ * Checks if a date is in the future. Uses the machine's local timezone.
```

Comment:
I don't understand the "uses the machine's local timezone" comment. Whether or not a date is in the future isn't timezone-dependent. You can see this for yourself in the [implementation of `isFuture`](https://github.com/date-fns/date-fns/blob/dd66398305c2b015fba3c1b3d31ccff42ee8d4cf/src/isFuture/index.ts#L23). It uses `Date.now()`, which is always in UTC.